spec/pdc/v1/global_component_contact_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+
+describe PDC do
+  before do
+    VCR.insert_cassette fixture_name
+  end
+
+  after do
+    VCR.eject_cassette
+  end
+
+  let(:contact) { PDC::V1::GlobalComponentContact }
+
+  it 'must has compose' do
+    contact.where(component: 'vlgothic-fonts').first.contact.must_be_instance_of OpenStruct
+  end
+
+  describe 'count' do
+    it 'must return number of resources' do
+      count = contact.count
+      count.must_equal 12546
+    end
+
+    it 'works with where' do
+      count = contact.where(component: 'vlgothic-fonts' ).count
+      count.must_equal 3 # as there is 3 fields per entry
+    end
+
+    it 'can find global contact by component name' do
+      count = contact.where(component: 'vlgothic-fonts').count
+      count.must_equal 3
+    end
+  end # count
```

Comment:
let's remove # count as it will give pronto error, don't know why it doesn't pop up error in travis ci.